### PR TITLE
sendHTTPRequest() Function using Python

### PR DESCRIPTION
**Pull request by dejmedus**
Should close appwrite/appwrite#4177

Added sendHTTPRequest function to examples/python/
 
I also changed the path names in the previous python examples so it will be easier to change directories. 
e.x. python/object-detection/README.md used to say ```cd python/convert_phone_number_to_country_name```

success
success
headers and body optional

error: missing url/method
url and method required

error: url/method are empty strings
url:method cannot be empty strings
url:method cannot be empty

error: url path leads no where
incorrect path

error: invalid url
invalid url

### CODE REVIEW

File: python/send-http-request/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    "success":true,
+    "response":
+    {
+        "headers":{},
+        "code":200,
+        "body":"{\"total\":27,\"countries\":[]}"
+    }
+}
+```
+
```
Comment (Meldiron) [inline python/send-http-request/README.md:33]: Let's please add an example of failed output (to showcase what that would look like)

File: python/send-http-request/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
```
Comment (Meldiron) [inline python/send-http-request/README.md:34]: Let's please re-add env variable section, and let's mention there are no variables required to run the function.

**Review by Meldiron — CHANGES_REQUESTED** [review]
Amazing PR! 🤩 Let's please address a few small comments in the README file.

**Review by Meldiron — APPROVED** [review]
Looks great 🎉

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**